### PR TITLE
Pop flickql (quinn with daniel)

### PR DESCRIPTION
**Pull request by qlalberta**
#14 
pair programming to finish the first api specific request function

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -20,6 +20,57 @@ app.use(bodyParser.urlencoded({extended: true}));
...
+TMDbQuery('movie/550/recommendations');
+
+function loadDB() {
+  client.query(`
+    CREATE TABLE IF NOT EXISTS
+    movies (
+      movie_id SERIAL PRIMARY KEY,
+      title VARCHAR(255) UNIQUE NOT NULL,
+      release_date DATE,
+      synopsis TEXT NOT NULL,
```
Comment (Tyler-206) [inline server.js]: This is called overview

Comment (Tyler-206) [reply server.js]: not synopsis

**Review by Tyler-206 — CHANGES_REQUESTED** [review]
change synopsis to overview

File: server.js [hunk 2]
```diff
@@ -20,6 +20,57 @@ app.use(bodyParser.urlencoded({extended: true}));
...
+    method: 'GET'
+  };
+
+request(options, function(err, res, body) {
+  let json = JSON.parse(body);
+  console.log(json);
+})
+}
+
+TMDbQuery('movie/550/recommendations');
```
Comment (squeekyfoot) [inline server.js]: TMDbQuery needs to be camel-cased ie: tmdbQuery  Functions that start with uppercase letters should be reserved for constructor functions. Also, the data being returned from this function is not going anywhere, it is being stored in variable 'options' but is not being used outside of that function.

**Review by squeekyfoot — APPROVED** [review]
Looks great. Thanks for matching the properties to the database properties.

**Review by Tyler-206 — APPROVED** [review]

**Pull request merged**